### ISSUE TITLE
ELF/aarch64-adrp-ldr-got.s is flaky

### ISSUE THREAD

**Issue by nico** [llvm/llvm-project#53469]
It failed once here: http://45.33.8.238/macm1/26613/step_10.txt

```
...
: 'RUN: at line 47';   /Users/thakis/src/llvm-project/out/gn/bin/llvm-objdump --no-show-raw-insn -d /Users/thakis/src/llvm-project/out/gn/obj/lld/test/ELF/Output/aarch64-adrp-ldr-got.s.tmp/unpaired |    /Users/thakis/src/llvm-project/out/gn/bin/FileCheck --check-prefix=UNPAIRED /Users/thakis/src/llvm-project/lld/test/ELF/aarch64-adrp-ldr-got.s
: 'RUN: at line 56';   /Users/thakis/src/llvm-project/out/gn/bin/ld.lld /Users/thakis/src/llvm-project/out/gn/obj/lld/test/ELF/Output/aarch64-adrp-ldr-got.s.tmp/lone-ldr.o -o /Users/thakis/src/llvm-project/out/gn/obj/lld/test/ELF/Output/aarch64-adrp-ldr-got.s.tmp/lone-ldr
: 'RUN: at line 57';   /Users/thakis/src/llvm-project/out/gn/bin/llvm-objdump --no-show-raw-insn -d /Users/thakis/src/llvm-project/out/gn/obj/lld/test/ELF/Output/aarch64-adrp-ldr-got.s.tmp/lone-ldr |    /Users/thakis/src/llvm-project/out/gn/bin/FileCheck --check-prefix=LONE-LDR /Users/thakis/src/llvm-project/lld/test/ELF/aarch64-adrp-ldr-got.s
--
Exit Code: 74


********************
Testing:  0.. 10.. 20.. 30.. 40.. 50.. 60.. 70.. 80.. 90.. 
********************
Failed Tests (1):
  lld :: ELF/aarch64-adrp-ldr-got.s
```

@alexander-shaposhnikov

**Comment by nico** [comment]
Already reported on https://reviews.llvm.org/D112063 and allegedly fixed in 0d71f2e097b94bd785bcadfe38cb0875f7756d78 (…which is marked as NFC though).

**Comment by MaskRay** [comment]
Fixed by f131d4d0d0c5f9a1feb416f8738b1c1f76f277ed by @alexander-shaposhnikov
